.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=c07187b4ac8b5cbba8b9da7ed99820ce5b5a38a6&env=prod&clientType=FLUENTUI)

_Diffs exceeded the limit allowed for comments. Please use the report link above to check your diffs._


<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>